src/loader/multi_loader.rs

```diff
@@ -0,0 +1,70 @@
+use crate::Loader;
+use fluent_bundle::FluentValue;
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+pub use unic_langid::LanguageIdentifier;
+
+/// A loader comprised of other loaders.
+///
+/// This loader allows for loaders with multiple sources to be used
+/// from a single one, instead of a multiple of them.
+pub struct MultiLoader {
+    pub loaders: Vec<Box<dyn Loader>>,
+}
+
+impl MultiLoader {
+    /// Creates a [`MultiLoader`] without any loaders.
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    /// Creates a [`MultiLoader`] from an iterator of loaders.
+    pub fn from_iter(iter: impl IntoIterator<Item = Box<dyn Loader>>) -> Self {
+        Self {
+            loaders: iter.into_iter().collect(),
+        }
+    }
+}
```

Comment:
I would add a few that allow people to add or remove loaders.

Reply:
Yeah, didn't want to implement the entirety of `Vec`/`VecDeque`'s interface, so I left the field public.